main.sh

```diff
@@ -26,7 +27,12 @@ Response=$(
     --url "$SERVER_URL/api-pub/v1/axe-watcher/gh/$COMMIT_SHA"
 )
 
-IssueCount=$(echo "$Response" | jq .last_run_violation_count)
+if [ -n "${ENABLE_A11Y_THRESHOLD+x}" ] && [ "$ENABLE_A11Y_THRESHOLD" = true ]; then
```

Comment:
`= true` doesn't do what you're thinking it does here. Should this entire condition be:

```suggestion
if [ "$ENABLE_A11Y_THRESHOLD" = "true" ]; then
```